skills/azure-cost-calculator/references/services/databases/database-migration.md

```
@@ -0,0 +1,85 @@
+---
+serviceName: Azure Database Migration Service
+category: databases
+aliases: [DMS, Database Migration, DB Migration Service]
+---
+
+# Azure Database Migration Service
+
+**Primary cost**: vCore hourly rate × 730 (Premium tier only — Standard is free)
+
+> **Trap**: Meter names (`4 vCore`, `8 vCore`, `16 vCore`) repeat across General Purpose Compute and Premium Compute products. Always filter by `productName` to select the correct tier.
+
```

markdownlint-cli2 (0.20.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)